app/services/importers/new_court_case.rb

```diff
@@ -24,7 +24,7 @@ def perform
     private
 
     def oscn_id
-      params['casemasterid'].first.to_i
+      params['casemasterID'].first.to_i
```

Comment:
did you mean to update this?

Reply:
Yea, the param name was wrong, needed to be casemasterID